include/snowflake/client.h

```diff
@@ -335,6 +336,12 @@ typedef struct SF_STMT {
     int64 total_fieldcount;
     int64 total_row_index;
     void *params;
+    void **name_list;
+    union
+    {
+        int64 names_len;
+        int64 idx_len;
+    };
```

Comment:
Since these are the same type, you can probably just merge them into 1 variable called `params_len`